cmd/opentelemetry-collector/app/exporter/span_writer_exporter.go

```diff
@@ -18,29 +18,29 @@ import (
 	"context"
 	"io"
 
+	"github.com/open-telemetry/opentelemetry-collector/component"
+	"github.com/open-telemetry/opentelemetry-collector/component/componenterror"
 	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
 	"github.com/open-telemetry/opentelemetry-collector/consumer/consumerdata"
 	"github.com/open-telemetry/opentelemetry-collector/consumer/consumererror"
-	"github.com/open-telemetry/opentelemetry-collector/exporter"
 	"github.com/open-telemetry/opentelemetry-collector/exporter/exporterhelper"
-	"github.com/open-telemetry/opentelemetry-collector/oterr"
 	jaegertranslator "github.com/open-telemetry/opentelemetry-collector/translator/trace/jaeger"
 
 	jaegerstorage "github.com/jaegertracing/jaeger/storage"
 	"github.com/jaegertracing/jaeger/storage/spanstore"
 )
 
-// NewSpanWriterExporter returns exporter.TraceExporter
-func NewSpanWriterExporter(config configmodels.Exporter, factory jaegerstorage.Factory) (exporter.TraceExporter, error) {
+// NewSpanWriterExporter returns component.TraceExporter
+func NewSpanWriterExporter(config configmodels.Exporter, factory jaegerstorage.Factory) (component.TraceExporterOld, error) {
```

Comment:
Using `TraceExporterOld` for now until `jaegertranslator.OCProtoToJaegerProto(td)` changes to the argument to `/internal/data.TraceData`